### ISSUE TITLE
The scrollback is not quite right for the current population

### ISSUE THREAD

**Issue by alexchally** [meatspaces/meatspace-chat-v2#92]
I think that we need to revise how scrollback works a bit so that people don't log into an empty channel quite so often.

Maybe just dropping the time limit so it is always the last 30 messages, or maybe increasing the time limit to an hour instead of 10 minutes. There may be other ways of dealing with the situation, and I would like to open up a discussion about it so that we can evaluate the options as a community.

**Comment by cloudEdwards** [comment]
I agree alex, I was going to bring up this issue myself. There seems the be a perception that it will be full of messages saying "where's everyone?"  But I don't think so. I think if, when you log in, you see other meats, you'll just say hi and try to interact.  I think we could try it out, adn see how it works. We could always put it back the way it was if it doesn't work.

**Comment by cloudEdwards** [comment]
I am trying to find where the code is that decides how long the scroll back is. Travis was saying it isn't in the meatspace-chat-v2 repo. But in a module it depends on.  I was looking at  node-meatspace-publico repo. It might be in there in the .js somewhere. Any ideas as to where to find it?

**Comment by TheAlphaNerd** [comment]
Potential solution... talk more?

**Comment by jasonrhodes** [comment]
@TheAlphaNerd +1
